unit_tests/edge_kernels/UnitTestScalarOpenEdge.C

```diff
@@ -0,0 +1,73 @@
+/*------------------------------------------------------------------------*/
+/*  Copyright 2019 National Renewable Energy Laboratory.                  */
+/*  This software is released under the license detailed                  */
+/*  in the file, LICENSE, which is located in the top-level Nalu          */
+/*  directory structure                                                   */
+/*------------------------------------------------------------------------*/
+
+#include "kernels/UnitTestKernelUtils.h"
+#include "UnitTestUtils.h"
+#include "UnitTestHelperObjects.h"
+
+#include "edge_kernels/ScalarEdgeOpenSolverAlg.h"
+
+#ifndef KOKKOS_ENABLE_CUDA
+namespace {
+namespace hex8_golds  {
+static constexpr double rhs[8] = {
+0, -20, -24.755282581476, 0, 0, -20, -22.795084971875, 0 
+};
+
+static constexpr double lhs[8][8] = {
+{0, 0, 0, 0, 0, 0, 0, 0, },
+{0, 10, 0, 0, 0, 0, 0, 0, },
+{0, 0, 10, 0, 0, 0, 0, 0, },
+{0, 0, 0, 0, 0, 0, 0, 0, },
+{0, 0, 0, 0, 0, 0, 0, 0, },
+{0, 0, 0, 0, 0, 10, 0, 0, },
+{0, 0, 0, 0, 0, 0, 10, 0, },
+{0, 0, 0, 0, 0, 0, 0, 0, },
+};
+}
+}
+#endif
+
+TEST_F(ScalarOpenEdgeKernelHex8Mesh, NGP_scalar_edge_open_solver_alg)
```

Comment:
There is already a `SSTKernelHex8Mesh` that provides all the fields necessary to test this, can we reuse that instead of creating this additional class just for this BC?